### PR TITLE
Slows down Atmos processing

### PR DESCRIPTION
**Pull request by TDcoolguy300**
:cl: TDcoolguy300 AKA Gregg
tweak: Slowed down Atmospheric processing from 5 ms per call to 10 ms per call
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
will stop the server from lagging, should be merged for the time being

**Comment by ghost** [comment]
This technically changes nothing, the MC should technically slow atmos down once it starts slowing down the server

File: code/controllers/subsystem/air.dm [hunk 1]
```diff
@@ -10,7 +10,7 @@ SUBSYSTEM_DEF(air)
 	name = "Atmospherics"
 	init_order = INIT_ORDER_AIR
 	priority = 20
-	wait = 5
+	wait = 10
```
Comment (ghost) [inline code/controllers/subsystem/air.dm]: You've made changes to /tg/ files, please mark these changes.

**All modifications to non-hippie files should be marked.**

- Multi line changes start with `// hippie start` and end with `// hippie end`
- You can put a messages with a change if it isn't obvious, like this: `// hippie start - reason`
  - Should generally be about the reason the change was made, what it was before, or what the change is
  - Multi-line messages should start with `// hippie start` and use `/* Multi line message here */` for the message itself
- Single line changes should have `// hippie` or `// hippie - reason`

If you need to mirror a file, or function into a hippie-specific file, please leave behind a comment stating where it went.

```
// hippie start - Mirrored this function in <file> for <reason>
bunch of shitcode here
// hippie stop
```

Once you mirror a file, please follow the above for marking your changes, this way we know what needs to be updated when a file has been mirrored.

**Review by ghost — DISMISSED** [review]

**Comment by bardofrage** [comment]
This needs to be approved quickly.

**Review by GuyonBroadway — APPROVED** [review]
Approving but I doubt it'll fix anything.

**Comment by ghost** [comment]
panic

**Comment by KayeArray** [comment]
lol

**Pull request merged**